eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServerV2.java

```diff
@@ -87,6 +87,10 @@ public ResponseEntity getVariantsByRegion(@PathVariable("regionId") String regio
                                                       annotationVepVersion,
                                               @RequestParam(name = "annot-vep-cache-version", required = false) String
                                                       annotationVepCacheVersion,
+                                              @RequestParam(value = "page", required = false, defaultValue = "0")
+                                                      Integer pageNumber,
+                                              @RequestParam(value = "size", required = false, defaultValue = "20")
```

Comment:
I was talking about renaming the external parameter `value = "size"` :joy:

Reply:
I have changed the external parameter name as well now :sweat_smile: